### ISSUE TITLE
Scimax on Windows - Which version?

### ISSUE THREAD

**Issue by OrionRandD** [jkitchin/scimax#496]
;;;;;;;;;;;;;;;;;;;;;;;;;;;;

;; Windows has always been the most difficult installation target, and it is the least maintained.
;; Install git (http://git-scm.com/download/win). Open a git bash terminal. Run this command.
;; bash -c "$(curl -fsSL https://raw.githubusercontent.com/jkitchin/scimax/master/install-scimax-win.sh)"
;; There is an emacs 25.2 binary in this repository for Windows. You should be able to run the scimax.sh command to launch scimax now.

;; Do you mean that if I install Emacs in Windows, using this method my Emacs version will be 25.2?

;;;;;;;;;;;;;;;;;;;;;;;;;;;;

**Comment by jkitchin** [comment]
In principle you should be able to install any modern emacs, I recommend at least version 28 now. I don't run any windows machines now, so I am not sure what the best one is anymore. The directions you quoted are from several years ago. Then you just git clone scimax and load the init file. 

this is probably not going to fully featured, scimax uses some utilities like grep here and there that are not usually installed in Windows, but I guess a lot of things would be ok.

**Comment by OrionRandD** [comment]
> In principle you should be able to install any modern emacs, I recommend at least version 28 now. I don't run any windows machines now, so I am not sure what the best one is anymore. The directions you quoted are from several years ago. Then you just git clone scimax and load the init file.
> 
> this is probably not going to fully featured, scimax uses some utilities like grep here and there that are not usually installed in Windows, but I guess a lot of things would be ok.

So, you mean that: 
1. I install Emacs for Windows, say, this version:
 https://ftp.gnu.org/gnu/emacs/windows/emacs-29/
2.

**Comment by OrionRandD** [comment]
> In principle you should be able to install any modern emacs, I recommend at least version 28 now. I don't run any windows machines now, so I am not sure what the best one is anymore. The directions you quoted are from several years ago. Then you just git clone scimax and load the init file.
> 
> this is probably not going to fully featured, scimax uses some utilities like grep here and there that are not usually installed in Windows, but I guess a lot of things would be ok.

So, you mean that: 
1. I install Emacs for Windows, say, this version:
 https://ftp.gnu.org/gnu/emacs/windows/emacs-29/
2. Then: I pull scimax and make the adjustments?

 Thx

**Comment by jkitchin** [comment]
yes. I am pretty sure that should work. The first time you run something like `emacs -q -l /path/to/scimax/init.el` it will download a bunch of packages from Melpa, and historically I had to run it a few times because it would hang or something, but eventually it would run normally.
